schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/SchemaNameVersionId.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2016 Hortonworks.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hortonworks.registries.schemaregistry;
+
+import com.google.common.base.Preconditions;
+
+/**
+ * This class represents schema, its name, version and unique id.
+ */
+public class SchemaNameVersionId {
```

Comment:
I guess we already have `SchemaVersionInfo`, more information can be added in that and used  instead of creating a new class.